Source/WebCore/crypto/keys/CryptoKeyHMAC.cpp

```diff
@@ -46,8 +46,7 @@ static size_t getKeyLengthFromHash(CryptoAlgorithmIdentifier hash)
     case CryptoAlgorithmIdentifier::SHA_512:
         return 1024;
     default:
-        ASSERT_NOT_REACHED();
```

Comment:
Should this be reachable?

Reply:
It should not. I have added a RELEASE_ASSERT now.